src/test/java/zd/zero/waifu/motivator/plugin/tools/ToolBoxKtTest.kt

```diff
@@ -0,0 +1,44 @@
+package zd.zero.waifu.motivator.plugin.tools
+
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import java.util.*
+
+internal class BestGirl(val name: String)
+class ToolBoxKtTest {
+
+    @Test
+    fun allOfShouldOnlyBePresentIfAllAreThere() {
+        Assertions.assertThat(
+            allOf(
+                BestGirl("Ryuko").toOptional(),
```

Comment:
You could start a world war III with this statement 😅

Reply:
![Facts](https://media.giphy.com/media/l7hw7lsRbSnQI/giphy.gif)